mix.exs

```diff
@@ -21,8 +21,8 @@ defmodule Schemata.Mixfile do
          "-Wunknown"
        ]
      ],
-     aliases: aliases,
-     deps: deps]
+     aliases: aliases(),
+     deps: deps()]
   end
```

Comment:
It was a controversial decision within the Elixir community with a passionate debate in the PR comments and on the mailing list.

The ability to leave off parens is another holdover from Ruby syntax and a lot of the Elixir devs who came from Ruby were vehemently opposed to the change. There was a very specific use case in which a bare function call would cause hard-to-diagnose issues and Jose was firm that avoiding those situations was more important than the syntactic sugar.

Note that remote function calls (i.e., `Module.function`) can still leave the parens off with no warning. Also, there is no warning for local functions which take arguments, although leaving them off in that case can lead to head-scratching precedence problems if you aren't careful.